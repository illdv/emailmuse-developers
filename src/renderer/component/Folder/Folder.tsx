import * as React from 'react';
import { bindActionCreators } from 'redux';
import { connect } from 'react-redux';

import { IFolderActions } from 'src/renderer/component/Folder/flux/actions';

interface IFolder {
  actions?: IFolderActions;
}

export const Folder: React.SFC<IFolder> = ({ actions }) => {
  const onRequest = () => {
    // actions.showModal.REQUEST({});
  };

  return (
    <div>
      Test
    </div>
  );
};

/*const mapStateToProps = state => ({

});*/

const mapDispatchToProps = dispatch => ({
  // actions: bindActionCreators({ createFolder }, dispatch),
});
export default connect(null, mapDispatchToProps)(Folder);
