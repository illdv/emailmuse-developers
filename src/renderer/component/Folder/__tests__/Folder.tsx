import * as React from 'react';
import { shallow } from 'enzyme';
import { Folder } from '../Folder';

describe('<Folder>', () => {
  const render = shallow(<Folder/>);
  test('Render UI', () => {
    expect(render).toMatchSnapshot();
  });
});
