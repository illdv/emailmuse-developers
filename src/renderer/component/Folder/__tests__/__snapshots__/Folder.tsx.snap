// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Folder> Render UI 1`] = `
<div>
  Test
</div>
`;
